FindOnPage.md

```diff
@@ -0,0 +1,653 @@
+
+
+  /// 
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT IsCaseSensitive([in] BOOL value);
+
+
+  /// Gets the `ShouldMatchWord` property.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT ShouldMatchWord([out, retval] BOOL* value);
+
+
+  /// 
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT ShouldMatchWord([in] BOOL value);
+
+
+
+}
+
+///  
+/// Interface providing methods to access the find operation functionalities in the CoreWebView2.
+/// 
+// MSOWNERS: core (maxwellmyers@microsoft.com)
+[uuid(c9a130ca-a807-549c-9d76-8e09ccee3973), object, pointer_default(unique)]
+interface ICoreWebView2Staging17 : IUnknown {
+  /// Retrieves the find operation interface for the current web view.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT Find([out, retval] ICoreWebView2StagingFind** value);
+
+
+
+}
+```
+
+
+### Setting Up Find Configuration with MIDL3
+
+### CoreWebView2 Find Configuration and Direction
+
+```csharp
+namespace Microsoft.Web.WebView2.Core
+{
+    public enum CoreWebView2FindDirection
+    {
+        Forward,
+        Backward
+    }
+
+    public class CoreWebView2FindConfiguration
+    {
+        public string FindTerm { get; set; }
+        public CoreWebView2FindDirection FindDirection { get; set; }
+        public bool IsCaseSensitive { get; set; }
+        public bool ShouldMatchWord { get; set; }
+    }
+}
+```
+
+### CoreWebView2 Find Interface
+
+```csharp
+namespace Microsoft.Web.WebView2.Core
+{
+    /// Specifies the direction of find Parameters.
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    enum CoreWebView2FindDirection
+    {
+        /// Specifies a forward find in the document.
+        Forward, 
+        /// Specifies a backwards find in the document.
+        Backward, 
+    };
+
+runtimeclass CoreWebView2FindConfiguration : [default]ICoreWebView2FindConfiguration {}
+
+    /// Interface defining the find configuration.
+    /// This interface provides the necessary methods and properties to configure a find operation.
+    [com_interface("staging=ICoreWebView2StagingFindConfiguration")]
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    interface ICoreWebView2FindConfiguration 
+    {
+        /// Gets or sets the find term used for the find operation. Returns the find term.
+        String FindTerm { get; set; };
+
+        /// Gets or sets the direction of the find operation (forward or backward). Returns the find direction.
+        CoreWebView2FindDirection FindDirection { get; set; };
+
+        /// Determines if the find operation is case sensitive. Returns TRUE if the find is case sensitive, FALSE otherwise.
+        /// When toggling case sensitivity, the behavior can vary by locale, which may be influenced by both the browser's UI locale and the document's language settings. The browser's UI locale         
```

Comment:
We can make this text and the next more straight forward. Is the document's language only determined by the HTML lang tag? If there's an MDN doc that describes how this works you can link to that.

```suggestion
        /// The locale used to determine case sensitivity is the document's language specified by the HTML lang attribute, or if unspecified then the WebView2's UI locale.
```